bofire/strategies/stepwise/transforms.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+import pandas as pd
+
+import bofire.data_models.strategies.stepwise.transforms as data_models
+from bofire.data_models.domain.api import Domain
+
+
+class Transform:
```

Comment:
I think we do not need here a distinction between data model and functional model, as the transform does not need any non standard dependecies, or? So one can just implement it for the datamodel. The same could be done also for the conditions, or?

We should also have a loo here: https://github.com/facebook/Ax/tree/main/ax/modelbridge/transforms and think of making the transforms more prominent, as transforms of the domain, the candidates and the experiments could be also useful in other scenarios. Have a look for this to the PR of @Osburg, where he transform the complete domain between bound of -1 and 1. Having this kinds of transform, that could also be chained behind it each other could make sense, they would always take in the domain, experiments and candidates and return the transformed versions, what do you think?

Reply:
Moved to data models as discussed.

Reply:
@jduerholt can you have a look at the invalid/valid spec-stuff? I did something wrong and don't really get it.